source/documentation/administration_guide/topics/Restoring_a_Backup_with_the_engine-backup_Command.adoc

```diff
@@ -5,6 +5,7 @@ Restoring a backup using the engine-backup command involves more steps than crea
 
 [IMPORTANT]
 ====
-Backups can only be restored to environments of the same major release as that of the backup. For example, a backup of a {virt-product-fullname} version 4.2 environment can only be restored to another {virt-product-fullname} version 4.2 environment. Starting with {virt-product-fullname} 4.4.7, when using the engine-backup command, the minor version of the {engine-name} (such as 4.4.8) must be later than or equal to the minor version of the backup (such as 4.4.7). 
+The version of the {virt-product-fullname} {engine-name} (such as 4.4.8) used to restore a backup must be later than or equal to the {virt-product-fullname} {engine-name} version (such as 4.4.7) used to create the backup.
+Starting with {virt-product-fullname} 4.4.7, this policy is strictly enforced by the engine-backup command. See link:{URL_virt_product_docs}{URL_format}upgrade_guide/index#Updating_the_Red_Hat_Virtualization_Manager_minor_updates[Updating the {virt-product-fullname} {engine-name}] in the _Upgrade Guide_.
```

Comment:
Why do you mention the upgrade guide? See also https://bugzilla.redhat.com/show_bug.cgi?id=1991171 , especially comment 14. If this sentence (pointing at the upgrade guide) is for addressing that comment 14, then I am not sure it's a perfect fit. It might be best to have a copy here (real copy or inclusion, as you wish). You can also point at e.g. section 3.2 of [1].

[1] https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html/installing_red_hat_virtualization_as_a_standalone_manager_with_local_databases/installing_the_red_hat_virtualization_manager_sm_localdb_deploy#Installing_RHEL_for_RHVM_SM_localDB_deploy

Reply:
I wasn't sure if it applies or not... removing the link from the Admin Guide file...